orderbook/usecase/orderbook_usecase_test.go

```diff
@@ -35,6 +35,101 @@ func TestOrderbookUsecaseTestSuite(t *testing.T) {
 	suite.Run(t, new(OrderbookUsecaseTestSuite))
 }
 
+// Creates a new osmomath.Dec from a string
+// Panics if the string is invalid
+func (s *OrderbookUsecaseTestSuite) newDecFromStr(str string) osmomath.Dec {
+	s.T().Helper()
+	dec, err := osmomath.NewDecFromStr(str)
+	s.Require().NoError(err)
+	return dec
+}
+
+func (s *OrderbookUsecaseTestSuite) newOrder() orderbookdomain.Order {
+	s.T().Helper()
+
+	return orderbookdomain.Order{
+		TickId:         1,
+		OrderId:        1,
+		OrderDirection: "bid",
+		Owner:          "owner1",
+		Quantity:       "1000",
+		PlacedQuantity: "1500",
+		Etas:           "500",
+		ClaimBounty:    "10",
+		PlacedAt:       "1634764800000",
+	}
+}
+
+func (s *OrderbookUsecaseTestSuite) newLimitOrder() orderbookdomain.LimitOrder {
```

Comment:
Ditto about helper

Reply:
Moved to [OrderbookTestHelper](https://github.com/osmosis-labs/sqs/pull/499/files#diff-5fa55bd6410efa835405589edaffbb41ba95f810344564647d2d711e8bc15b67R123), please let me does it make sense.